src/main/java/blue/lapis/pore/impl/entity/PoreLivingEntity.java

```diff
@@ -265,66 +267,75 @@ public void removePotionEffect(PotionEffectType type) {
 
     @Override
     public boolean hasLineOfSight(Entity other) {
-        throw new NotImplementedException(); //TODO
+        throw new NotImplementedException(); // TODO
     }
 
     @Override
     public boolean getRemoveWhenFarAway() {
-        throw new NotImplementedException(); //TODO
+        throw new NotImplementedException(); // TODO
     }
 
     @Override
     public void setRemoveWhenFarAway(boolean remove) {
-        throw new NotImplementedException(); //TODO
+        throw new NotImplementedException(); // TODO
     }
 
     @Override
     public EntityEquipment getEquipment() {
-        throw new NotImplementedException(); //TODO
+        throw new NotImplementedException(); // TODO
     }
 
     @Override
     public void setCanPickupItems(boolean pickup) {
-        if (getHandle() instanceof Agent) {
```

Comment:
This should throw a NotImplementedException if the implementation is missing or impossible.